wfe/wfe.go

```diff
@@ -2079,6 +2085,8 @@ func (wfe *WebFrontEndImpl) Authz(
 				response)
 			return
 		}
+
+		response.Header().Add("Retry-After", strconv.Itoa(wfe.retryAfterAuthz))
```

Comment:
Similar comment here as above: the Retry-After header is only really relevant when the Authz is in the "pending" state; consider checking for `authz.Status == acme.StatusPending` and only adding the header in that case.

Reply:
@aarongable since authorizations begin life in the `pending` state, should it also check that at least one of the challenges is undergoing validation?

Otherwise Pebble will send a `Retry-After` on a brand new authorization, where the client hasn't responded to any of the challenges, and there is nothing to wait for.

This will probably require implementing the `processing` state on challenges, or an internal flag on the challenge object, though.